Makefile

```diff
@@ -15,10 +18,15 @@ build:
 debug:
 	go run -race $(LDFLAGS) .
 
+.PHONY: lint
+lint:
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.53.3
```

Comment:
If I were to run `make lint` on my machine, this line forces an install potentially over an already-installed (and more up-to-date, in future) version of this app on my machine. Not ideal.

I would remove this line and - instead - put it into a file such as `ci/scripts/lint.sh` which (along with `ci/lint.yml`) appears to be missing - did you forget to add them? - see [other repo](https://github.com/ONSdigital/dp-upload-service/blob/develop/ci/scripts/lint.sh#L4) for an example.

Therefore, the linter is installed when it's needed.